rust/kernel/prelude.rs

```diff
@@ -2,25 +2,27 @@
 
 //! The `kernel` prelude.
 //!
-//! These are most common items used by Rust code in the kernel, intended to
-//! be imported by all Rust code, for convenience.
+//! These are the most common items used by Rust code in the kernel,
+//! intended to be imported by all Rust code, for convenience.
 //!
 //! # Examples
 //!
 //! ```
 //! use kernel::prelude::*;
 //! ```
 
-pub use alloc::{borrow::ToOwned, string::String};
+pub use core::pin::Pin;
 
-pub use super::build_assert;
+pub use alloc::{borrow::ToOwned, boxed::Box, string::String, sync::Arc, vec::Vec};
```

Comment:
Do we still use `ToOwned` at all after fallible alloc? I guess all implementations are removed now?

Reply:
Yeah, all users are gone now. Let me add a commit.